zetasql/civetweb-libdir.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/bazel/civetweb.BUILD b/bazel/civetweb.BUILD
```

Comment:
We should also remove the `zetasql` folder.

Reply:
done!